app/views/posts/show.html.erb

```diff
@@ -1,2 +1,22 @@
-<h1>Post ID <%= @post.id %> of User <%= @user.name %></h1>
-<li><%= @post.title %></li>
\ No newline at end of file
+<div class="individual-post-container">
+  <div class="post-top-bar">
+    <h2>Post <%= "#" + @post.id.to_s %> by <%= @user.name %></h2>
+        <div class="post-comments-likes">
+          <span>Comments: <%= @post.comments_counter %>, </span>
+          <span>Likes: <%= @post.likes_counter %></span>
+        </div>
+  </div>
+  <p><%= @post.text %></p>
+</div>
+<div class="comment-box">
+  <% @post.recent_five.each do |comment| %>
+    <div class="each-commet">
+      <span><%= comment.author.name %>: </span>
+      <span><%= comment.text %></span>
+    </div>
+  <% end %>
```

Comment:
- [ ] Please make sure that post show page displays post with all comments listed as shown on the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_single_post.png) not only 5 comments.

Current | Expected 
---| ---|
![image](https://github.com/misalazeem/Blog-App-Rails/assets/106119105/d1b1577d-36fc-420f-9e41-794d26d7b1b6) | ![image](https://github.com/misalazeem/Blog-App-Rails/assets/106119105/01e48e84-556c-4e79-8df4-2e93e3111fd8)